network_analysis/routing-around-barriers/src/main/resources/style.css

```diff
@@ -1,27 +1,3 @@
-.panel-region .label {
-  -fx-text-fill: white;
-}
-
-.label {
-  -fx-text-fill: black;
-}
-
-.slider .axis {
-    -fx-tick-label-fill: white;
-}
-
-.range-slider .axis {
-    -fx-tick-label-fill: white;
-}
-
-.panel-region .check-box {
-   -fx-text-fill: white;
-}
-
 .panel-region .radio-button {
    -fx-text-fill: white;
```

Comment:
is this change for Routing Around Barriers intended for this PR? We should probably stick to making changes only to one sample at a time.